themes/ansible-community/sass/main.scss

```diff
@@ -20,6 +20,7 @@
 @import "homepage-bullhorn-band";
 @import "homepage-events-band";
 @import "homepage-platform-band";
+@import "ecosystem";
```

Comment:
I want to get a bit more knowledgeable about the website so taking a moment to ask the goober question here - why have a separate style sheet just for the ecosystem page? Wouldn't it help to set the same style for anything that uses buttons like this for example so we know they all stay the same on sub pages like this?

Reply:
Separate style sheets give more granular control so you don't make a change on one page that inadvertently changes styling on another page. It is good to try and have central, reusable components and we do that for buttons on the homepage here: https://github.com/ansible-community/community-website/blob/main/themes/ansible-community/sass/_homepage-clickable.scss

In the case of the ecosystem page the buttons are different to the homepage. Notice that they are "inline-block" instead of "flex" (which is inherited on the homepage).

I guess there are some properties that we can inherit from the homepage buttons but that can get tricky really fast. So I thought it would just be more straightforward to leave the ecosystem page as a unique thing in case someone else down the line needs to make a change. There's not as much complexity and you need to focus on one page instead of multiple pages, even though it does result in more properties.

Should we need to create another secondary page with buttons, it's entirely feasible to break the button styling out into a shared scss definition. To my knowledge there are no plans for that so I didn't bother to keep things self-contained and straightforward.